README.md

```diff
@@ -66,6 +66,7 @@ If you are not on mac, please do this extra step and install these libraries as
 npm install --save-dev @nomiclabs/hardhat-waffle ethereum-waffle chai @nomiclabs/hardhat-ethers ethers
```

Comment:
This line needs to change to `npm install --save-dev @nomicfoundation/hardhat-toolbox`